### ISSUE TITLE
Empty reply from server when converting an existing table to WAL

### ISSUE THREAD

**Issue by 180254** [questdb/questdb#2962]
### Describe the bug

When 'alter table ABC set type WAL;' is executed, the server closes the connection with no response.

### To reproduce

1. Go to Web Console
2. Execute 'alter table ABC set type WAL;'
3. The server closes the connection with no response, resulting in:
- "An error occured, please try again" message is displayed
- The status is changed to "Error connecting to QuestDB"
- The list of tables is changed to "Cannot load tables An error occured, please try again"

### Expected Behavior

The operation is successful.
The server responds with some kind of confirmation.
WebConsole does not display an error message.

### Environment

```markdown
- **QuestDB version**: questdb/questdb:7.0.0 (docker image)
- **OS**: Ubuntu 22.04.1 LTS, Docker version 20.10.22
- **Browser**: Firefox 109.0, curl 7.68.0
```

### Additional context

The curl log is as follows:

```
curl -v 'http://192.168.50.21:9000/exec?limit=0%2C1000&explain=true&count=true&src=con&query=alter%20table%20some_meter1%20set%20type%20WAL%3B%0A&timings=true' -H 'User-Agent: xx' -H 'Accept: */*' -H 'Accept-Language: en-US,en;q=0.5' -H 'Accept-Encoding: gzip, deflate' -H 'Referer: http://192.168.50.21:9000/' -H 'Connection: keep-alive' -H 'Cookie: xx'
*   Trying 192.168.50.21:9000...
* TCP_NODELAY set
* Connected to 192.168.50.21 (192.168.50.21) port 9000 (#0)
> GET /exec?limit=0%2C1000&explain=true&count=true&src=con&query=alter%20table%20some_meter1%20set%20type%20WAL%3B%0A&timings=true HTTP/1.1
> Host: 192.168.50.21:9000
> User-Agent: xx
> Accept: */*
> Accept-Language: en-US,en;q=0.5
> Accept-Encoding: gzip, deflate
> Referer: http://192.168.50.21:9000/
> Connection: keep-alive
> Cookie: xx
> 
* Empty reply from server
* Connection #0 to host 192.168.50.21 left intact
curl: (52) Empty reply from server
```

**Comment by glasstiger** [comment]
Fix is in [#2963](https://github.com/questdb/questdb/pull/2963) and it will require a release.

**Comment by glasstiger** [comment]
btw, while the fix gets released, if you have a postgres client you can issue the ALTER command via that.
The postgres interface should work.
